### PR TITLE
Move popup config into a dict field

### PR DESCRIPTION
**Pull request by CFarcy**

### CODE REVIEW

File: terra_layer/migrations/0051_auto_20201014_1348.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.1.2 on 2020-10-14 13:48
```
Comment (jrmi) [inline terra_layer/migrations/0051_auto_20201014_1348.py:1]: Why it seems to be an automated migration (see file name) but it's a manual data migration

File: terra_layer/tests/test_views.py [hunk 2]
```diff
@@ -309,7 +309,7 @@ def test_layer_view_with_custom_style(self):
                 },
             ],
             minisheet_enable=True,
-            popup_enable=True,
+            popup_config={"enabled": True},
```
Comment (jrmi) [inline terra_layer/tests/test_views.py:312]: enable vs enabled ? Check consistency.

**Review by jrmi — CHANGES_REQUESTED** [review]
Some minor things to fix.

File: terra_layer/migrations/0051_auto_20201014_1348.py (hunk 1 above)
Comment (CFarcy) [reply terra_layer/migrations/0051_auto_20201014_1348.py:1]: I create an empty migration with `python manage.py makemigrations --empty`. I followed the documentation for data migration https://docs.djangoproject.com/fr/2.2/topics/migrations/#data-migrations

File: terra_layer/tests/test_views.py (hunk 2 above)
Comment (CFarcy) [reply terra_layer/tests/test_views.py]: I changed it. It is now `enable` instead of `enabled`

**Review by jrmi — APPROVED** [review]

**Pull request merged**